src/components/shared/MarkerContextMenu.js

```diff
@@ -37,8 +38,7 @@ import { getMarkerFullDescription } from '../../profile-logic/marker-data';
 type StateProps = {|
   +previewSelection: PreviewSelection,
   +committedRange: StartEndRange,
-  +selectedMarker: Marker | null,
-  +thread: Thread,
+  +rightClickedMarker: RightClickedMarkerInfo | null,
```

Comment:
nit: maybe call this property `rightClickedMarkerInfo`. In this project we try to have very descriptive names, even if this makes them long.